siac/main.go

```diff
@@ -29,8 +28,13 @@ func apiGet(call string) (*http.Response, error) {
 	if host, port, _ := net.SplitHostPort(addr); host == "" {
 		addr = net.JoinHostPort("localhost", port)
 	}
-
-	resp, err := http.Get("http://" + addr + call)
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "http://"+addr+call, nil)
+	if err != nil {
+		return nil, errors.New("no response from daemon")
```

Comment:
Hmm, there's still a chance of a developer mistake down the line that results in malformed urls. I'm not sure I agree that the error should not be checked.